method: org.apache.fop.render.afp.exceptions.NestedRuntimeException.printStackTrace:1332000268
478889015:this := @this: org.apache.fop.render.afp.exceptions.NestedRuntimeException
2066449796:pw := @parameter0: java.io.PrintWriter
1019490159:$r0 = this.<org.apache.fop.render.afp.exceptions.NestedRuntimeException: java.lang.Throwable underlyingException>
1946566547:if $r0 != null goto virtualinvoke pw.<java.io.PrintWriter: void println(java.lang.Object)>(this)
1975702757:specialinvoke this.<java.lang.RuntimeException: void printStackTrace(java.io.PrintWriter)>(pw)
1244086985:goto [?= return]
299729596:virtualinvoke pw.<java.io.PrintWriter: void println(java.lang.Object)>(this)
212353508:$r0 = this.<org.apache.fop.render.afp.exceptions.NestedRuntimeException: java.lang.Throwable underlyingException>
1603659522:virtualinvoke $r0.<java.lang.Throwable: void printStackTrace(java.io.PrintWriter)>(pw)
1785816404:return
